blog/2024-06-05.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Getting Started on NEAR Using Windows
+authors: [flmel]
+slug: getting-started-on-windows
+tags: [windows, tutorial, getting-started]
+hide_table_of_contents: true
+---
+In this article, we will cover how to install WSL and setup a NEAR development environment on Windows.
+
+<!-- truncate -->
+
+### WSL
+
+WSL, or Windows Subsystem for Linux, is a compatibility layer for running Linux binary executables natively on Windows. It allows us to run a GNU/Linux environment directly on Windows without the overhead of a traditional virtual machine or dual-boot setup.
+
+#### Enable WSL from `Windows Features`
+
+`Control Panel > Programs > Turn Windows Features on or off`  
+Scrolling down will reveal the option `Windows Subsystem for Linux`. Make sure it is enabled and press OK to confirm (you will be asked to reboot your computer).
+
+![Windows Features WSL Enabled](../website/static/docs/blog/windows-features-wsl-enabled.jpg)
+
+#### Start WSL and Install Ubuntu
+
+Now we will spend some time in either `PowerShell` or [Windows Terminal](https://aka.ms/terminal), which is a modern terminal application that supports various command-line tools and shells.
+
+```
+wsl --install --d Ubuntu
+```
+
+**Read more on WSL:**  
+https://learn.microsoft.com/en-us/windows/wsl/install  
+https://learn.microsoft.com/en-us/windows/wsl/setup/environment  
+https://learn.microsoft.com/en-us/windows/wsl/
+
+### Install Required Packages
+
+```bash
+apt-get update
+apt-get install gcc make libudev-dev openssl pkg-config unzip -y
+```
+
+:::tip Permission Denied?
+Depending on your setup, you might need to run the commands with `sudo`. This temporarily grants your user elevated privileges to perform tasks that require higher permissions.
+:::
+
+### Setup Developer Environment
+
+Now that we have WSL enabled and running Ubuntu, NEAR currently supports two SDKs ([JavaScript/TypeScript](https://github.com/near/near-sdk-js) and [Rust](https://github.com/near/near-sdk-rs)). Their requirements are different, but we can install both if we choose.
+
+#### Node (JS/TS)
+
+Node.js is a JavaScript runtime environment that executes code outside a web browser, enabling the development of server-side applications. In NEAR development, it can be used to write smart contracts in [TypeScript](https://www.typescriptlang.org/).
```

Comment:
```suggestion
Node.js is a JavaScript runtime environment that executes code outside a web browser, enabling the development of server-side applications. In NEAR development, it can be used to write smart contracts in [TypeScript](https://www.typescriptlang.org/), as well as to create Web applications that interact with NEAR.
```